internet-programming/client/Client.java

```diff
@@ -0,0 +1,47 @@
+package client;
+import java.io.*;
+import java.net.*;
+
+public class Client {
+    public static void main(String[] args) {
+        try (Socket socket = new Socket("localhost", 31337);
+             BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
+             PrintWriter out = new PrintWriter(socket.getOutputStream(), true);
+             BufferedReader inputBuff = new BufferedReader(new InputStreamReader(System.in))) {
+
+            System.out.println("Connected to server");
+
+            String request;
+            while (true) {
+                System.out.println("Enter request (ADD, VIEW, AVERAGE) or 'EXIT' to quit:");
+                request = inputBuff.readLine();
+
+                if (request.equalsIgnoreCase("EXIT")) {
+                    out.println("DISCONNECT"); 
+                    break;
+                }
+
+                if (request.equalsIgnoreCase("ADD")) {
+                    System.out.println("Enter student data in the format 'name,grade1,grade2,...'");
```

Comment:
I believe that case sensitivity is not necessary for the functions, especially since they are called from the console. Working with case sensitivity can add unnecessary complexity.

Reply:
I'm sorry, I didn't express myself correctly.It's good to use equalsIgnoreCase and is ok just you just can reverse the check "ADD".equalsIgnoreCase(request) and it's better to use switch  than ifs.If feasible, prefer switch expressions over if/else statements for improved readability